sky/cli.py

```diff
@@ -1120,7 +1120,7 @@ def queue(clusters: Tuple[str], skip_finished: bool, all_users: bool):
         _show_job_queue_on_cluster(cluster, handle, backend, code)
 
     for local_cluster in local_clusters:
-        if local_cluster not in clusters:
+        if local_cluster not in clusters and len(clusters) > 1:
```

Comment:
Mind explaining this change?  What happens if I have say clusters A, B, C which are all local + uninitialized, and I do `sky queue A B` (or `sky queue my-cloud-clus A`)? Would I see the message "... C is uninitialized?"

Reply:
Fixed it, `C is uninitialized` should not show when multiple clusters are specified. These messages will only appear when `sky queue` is specified without specific clusters.